packages/cli/src/lib/implementation/core-config.integration.test.ts

```diff
@@ -137,4 +143,16 @@ describe('parsing values from CLI and middleware', () => {
       outputDir: 'cli-outputdir',
     });
   });
+
+  it('should fill categories if not given in config file', async () => {
```

Comment:
The word "fill" somewhat implies we have some meaningful defaults to me, I'd more explicit that it's just `undefined` :arrow_right: `[]`.

```suggestion
  it('should set empty array if categories not given in config file', async () => {
```

Reply:
Fully agree.